details/structured-groups.md

```diff
@@ -0,0 +1,85 @@
+# Structured groups
+
+Group ID represents a group/roles concept in the ROR ACL. You can define a neat, human-readable ID of the group when you introduce groups on the configuration level.
+However, when group IDs come from external integrations, the group IDs may have an unfriendly format.
+That's why we introduced a structured group concept, which consists of:
+* Group ID - a group identifier, used in the `groups`, `groups_or`, and `groups_and` ACL rules
+* Group Name - a human-readable name of the group. If not defined, it's equal to the group ID.
+
+Thanks to the structured groups, you can configure neat group names and hide external integrations internals from end users(ReadonlyREST Enterprise Kibana plugin uses group names in the Tenancy Selector)
```

Comment:
maybe we should say it at the beginning that the feature will be useful in the context of Kibana Enterprise. For Pro and Free users it's not so relevant, right?